### PR TITLE
Fix install

### PR DESCRIPTION
**Pull request by FabianWilkens**
## Problem

- Install not possible
- Is public help text missing

## Solution

- Fix install problem in install and upgrade script
- Remove is_public option

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by FabianWilkens** [comment]
!testmw

**Comment by FabianWilkens** [comment]
!testme

File: check_process [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		setup_sub_dir=1
 		setup_root=1
 		setup_nourl=0
-		setup_private=1
+		setup_private=0
```
Comment (yalh76) [inline check_process:16]: no need to remove private test, it's an usual issue we have those days on ci-apps-dev

Comment (FabianWilkens) [reply check_process:16]: Oh really?
I removed the private check, because this app is now always public.
This app in private makes no sense to me.

Comment (yalh76) [reply check_process:16]: ok then, It was just to let you know ^^
https://github.com/YunoHost/package_check/issues/123

**Comment by FabianWilkens** [comment]
Test was successfully on my onw ci.
2022-05-26 22_21_45-YunoRunner for CI - Firefox Developer Edition

**Pull request merged**